Doc/library/stdtypes.rst

```diff
@@ -84,12 +84,12 @@ These are the Boolean operations, ordered by ascending priority:
 +-------------+---------------------------------+-------+
 | Operation   | Result                          | Notes |
 +=============+=================================+=======+
-| ``x or y``  | if *x* is false, then *y*, else | \(1)  |
-|             | *x*                             |       |
-+-------------+---------------------------------+-------+
-| ``x and y`` | if *x* is false, then *x*, else | \(2)  |
+| ``x or y``  | if *x* is True, then *x*, else  | \(1)  |
 |             | *y*                             |       |
 +-------------+---------------------------------+-------+
+| ``x and y`` | if *x* is True and *y* is True, | \(2)  |
+|             | then True, else False           |       |
```

Comment:
I think you misunderstood the other comment, as this now doesn't cover something like

```
>>> [] and [1]
[]
```

Reply:
could you please tell then how should it be framed

Reply:
`if x and y, then True, else False`

it covers if it's Null, empty, False...

Reply:
thanks i'll do it

Reply:
The original mention of 'true'/'false' refers to all truthy and falsey values, which include 0, None, empty lists, etc., and saying `True` excludes those as `True` refers to just the boolean value. I also don't think that it's all that helpful for a table to have an entry to say `x and y` then its explanation also say `x and y`, so I would actually recommend just sticking with the original for this `and` section.